package.json

```diff
@@ -26,7 +26,8 @@
       "CRM",
       "angular",
       "inject",
-      "moment"
+      "moment",
+      "SimpleBar"
```

Comment:
this is an overkill, `SimpleBar` is used only once, in one file. Declare it global in the file directly
```js
/* global SimpleBar */
```